x-pack/plugins/apm/public/components/app/correlations/error_correlations.tsx

```diff
@@ -63,10 +63,11 @@ export function ErrorCorrelations({ onClose }: Props) {
     `apm.correlations.errors.fields:${serviceName}`,
     defaultFieldNames
   );
+  const hasFieldsNames = fieldNames.length > 0;
```

Comment:
nit: Shouldn't it be:
```suggestion
  const hasFieldNames = fieldNames.length > 0;
```
?